modules/tinymce/src/core/main/ts/api/Mode.ts

```diff
@@ -82,14 +81,17 @@ export const create = (editor: Editor): EditorMode => {
       editorReadOnly: true
     }
   });
+  const activeMode = Cell<[ string, EditorModeApi ]>([ 'design', availableModes.get().design ]);
```

Comment:
I believe the second element of the array is no longer needed since we made selection enabled the default?

The only place it's used now is in `switchToMode` and if you look at the diff, it already has that information available.

- [ ] revert back to just a string